rfd/0054-passwordless-macos.md

```diff
@@ -295,13 +281,13 @@ and [Passwordless RFDs][passwordless rfd].
 UX is discussed throughout the design, but here is a summary of changes:
 
 `tsh login --proxy=example.com` will automatically do passwordless Touch ID
-login, if possible (server allows passwordless, appropriate hardware exists, one
-credential registered for "example.com")
+login, if appropriate (server allows passwordless, hardware present, credential
+registered for "example.com")
```

Comment:
This is an interesting one, because we could easily go for Touch ID passwordless here as long as the cluster doesn't forbid it (even it password+MFA is the default).

Thoughts @klizhentas @xinding33 ?

Reply:
I agree. I don't see any downside in supporting passwordless by default.